frankenphp.go

```diff
@@ -517,25 +502,25 @@ func ServeHTTP(responseWriter http.ResponseWriter, request *http.Request) error
 }
 
 //export go_handle_request
-func go_handle_request() bool {
+func go_handle_request(threadIndex int) bool {
```

Comment:
Shouldn't we use `uint64_t` or `uint32_t` everywhere? (C-side and Go side)? The current implementation may trigger incompatibilities between C and Go types if a (very) large number of threads is started.

```suggestion
func go_handle_request(threadIndex C.uint64_t) bool {
```

Reply:
I was wondering how go treats the integers coming from C since they don't seem to need casting. I changed it to `uint32_t` since I'm not sure how `uint64_t` would be cast on 32 bit systems.

Reply:
After thinking about this a bit more, maybe the best option is `C.uintptr_t`. As Go's `int` type, it will have a different size depending on the platform capabilities, and it will prevent a conversion C-side.

Reply:
Should it then be a map of `C.uintptr_t`? A slice still works, but it somehow feels weird to do this: `phpThreads[C.uintptr_t(i)]`

Reply:
it shouldn't be an issue